packages/web-components/src/components/va-button/va-button.tsx

```diff
@@ -50,6 +53,22 @@ export class VaButton {
    */
   @Prop({ reflect: true }) disabled?: boolean = false;
 
+  /**
+   * If `true`, the click event will not fire.
+   */
+  @Prop({ reflect: true }) loading?: boolean = false;
+
+  @Watch('loading')
+  announceLoadingChange(newValue: boolean, oldValue: boolean) {
+    if (oldValue && !newValue) {
+      var me = this;
+      this.showCompletedMessage = true;
+      setTimeout(() => {
+        me.showCompletedMessage = false;
+      }, 3000);
```

Comment:
I'm not sure I understand the need for a timeout and why this specific time is used. Do you mind clarifying?

Reply:
The `<span role="status">` will only be read by the screen reader if there are changes to the content of the span. This is the reasoning for the span always being rendered, even if the loading prop is false, because if we render it when the prop changes the reader will not count that as a change and "Loading" will never be read.

The same applies to this timeout. If there is not any time between when the completion message is set and when it is set to null, the reader won't detect the change and the message won't be read. Its an arbitrary number to guarantee the "Loading complete" is read.

Reply:
That makes sense. Thanks for the explanation!